packages/playwright/src/matchers/expect.ts

```diff
@@ -379,6 +409,7 @@ async function pollMatcher(qualifiedMatcherName: string, info: ExpectMetaInfo, p
       let matchers = createMatchers(value, innerInfo, prefix);
       if (info.isNot)
         matchers = matchers.not;
+      setMatcherCallContext({ expectInfo: info, testInfo });
```

Comment:
This will be called in `ExpectMetaInfoProxyHandler`, why do we need it here too?